apps/web/src/components/open-chat-component.tsx

```
@@ -0,0 +1,701 @@
+          <Reasoning
+            key={`${message.id}-${index}`}
+            className="w-full"
+            isStreaming={status === 'streaming' && index === message.parts.length - 1 && message.id === rawMessages.at(-1)?.id}
+          >
+            <ReasoningTrigger />
+            <ReasoningContent>{part.text}</ReasoningContent>
+          </Reasoning>
+        );
+
+      case 'dynamic-tool':
+        const toolPart = part as DynamicToolUIPart;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)